src/main/java/io/cryostat/recordings/RecordingArchiveHelper.java

```diff
@@ -293,9 +293,10 @@ private Path searchSubdirectories(
         return null;
     }
 
-    private String writeRecordingToDestination(
+    public String writeRecordingToDestination(
             JFRConnection connection, IRecordingDescriptor descriptor)
-            throws IOException, URISyntaxException, FlightRecorderException, Exception {
+            throws IOException, URISyntaxException, FlightRecorderException,
+                    EmptyRecordingException, Exception {
```

Comment:
Since we're here, can this `throws` declaration be cleaned up? In particular, it would be nice to remove the `Exception` from the end, since there are plenty of specific types here too. `EmptyRecordingException` is also a subclass of `IOException`, so I don't think it needs to be mentioned here necessarily.

Reply:
It seems like `connection.getService()` could throw an `Exception`. Should we keep `Exception` in the method declaration? 
![image](https://user-images.githubusercontent.com/84587295/129212578-a165a8fa-6135-48c2-8e53-841aa9d545df.png)